### ISSUE TITLE
Verify test breaks on release branch after each version bump

### ISSUE THREAD

**Issue by enisoc** [kubernetes/release#326]
The [version bump](https://github.com/kubernetes/kubernetes/commit/00fb653e406a4b5bbac71a10c3e330afef960f13) committed on the release branch by anago breaks `make verify`.

To fix it, I have to run `hack/update-all.sh` afterward. Probably anago should run this after bumping the version, before committing the change.

**Comment by david-mcmahon** [comment]
That seems like a new requirement.  Did this not come up during the last release on the release-1.6 branch?  Do you have any details/logs on what exactly failed?

**Comment by enisoc** [comment]
It came up during the 1.6 release, but I didn't know it was unexpected. I just always ran update-all after a release cut because the fast-forward script did it. If this is new, I guess it's because update/verify changes/checks some things it didn't used to.

Gubernator is not loading for me right now, but this URL should contain an example:

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/45749/pull-kubernetes-verify/29935/

Basically there are things generated by the update script that change when the version is bumped, and the verify script notices that the generated stuff doesn't match because it contains the old version.

**Comment by enisoc** [comment]
This seems to have come back:

https://github.com/kubernetes/kubernetes/pull/52526
https://github.com/kubernetes/kubernetes/pull/52559

Could you take a look?

**Comment by david-mcmahon** [comment]
@enisoc apologies.  [The (latest) fix](https://github.com/kubernetes/release/pull/408) landed earlier today but after those releases.
